examples/advanced_operations/add_app_campaign.py

```diff
@@ -252,9 +258,22 @@ def _create_app_ad(client, customer_id, campaign_resource_name):
         customer_id, [ad_group_operation])
 
     ad_group_resource_name = ad_group_response.results[0].resource_name
-    print(f'Ad Group created {ad_group_resource_name}.')
+    print(f'Ad Group created with resource name: "{ad_group_resource_name}".')
+    return ad_group_resource_name
+
+
+def _create_app_ad(client, customer_id, ad_group_resource_name):
+    """Creates an App ad for a given ad group.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID str.
+        ad_group_resource_name: the ad group where the ad will be added.
 
-    # Create the ad group ad
+    Returns:
+        None.
```

Comment:
We probably don't need this `Returns` doc, or the return statement at the end of the function, since it'll return `None` anyway. I know it's not part of the changes but we might as well clean it up here.

Reply:
fixed.